.eslintrc

```diff
@@ -1,33 +1,24 @@
 {
-  "rules": {
-    "indent": [2, 2, {
-      "VariableDeclarator": {
-        "var": 2,
-        "let": 2,
-        "const": 3
-      }
-    }],
-    "no-empty": [2],
-    "no-console": [0],
-    "comma-dangle": [2],
-    "no-unused-vars": [0],
-    "linebreak-style": [2, "unix"],
-    "semi": [2, "always"]
-  },
+  "parser": "babel-eslint",
+  "extends": "./node_modules/mofo-style/.eslintrc.yaml",
   "env": {
+    "node": true,
     "commonjs": true,
     "es6": true,
     "browser": true
   },
-  "extends": "eslint:recommended",
   "parserOptions": {
+    "ecmaVersion": 6,
     "ecmaFeatures": {
-      "es6": true,
-      "jsx": true,
-      "experimentalObjectRestSpread": true
+      "jsx": true
     }
   },
+  "rules": {
+    "no-console": 1,
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1
```

Comment:
ummm I was getting these errors (see below)

<img width="531" alt="screen shot 2016-02-22 at 10 14 18 am" src="https://cloud.githubusercontent.com/assets/2896608/13228901/19791846-d953-11e5-9f99-c85188a03569.png">

and `eslint-plugin-react` suggests to use this `jsx-uses-vars` rule. Setting it to `1` okay?